### ISSUE TITLE
Active cases at the top doesn't match active cases in the region

### ISSUE THREAD

**Issue by mikekchar** [reustle/covid19japan#504]
I'm curious why the number of active cases at the top of the page doesn't match the active cases in the regions.  My naive thought was that I should be able to add up the active cases in the regions and get the active cases for the country, but it seems to be *very* different.  For example today the active cases is 1,007 while the active cases for Kansai alone is 2,873.  Is it a difference in information sources or are they referring to different things?

**Comment by liquidx** [comment]
Yes, you're spotting an issue where the data sources we're using are not agreeing. We get our confirmed cases numbers from NHK and our recovery numbers from MHLW. However, for some of the prefectures (notably Tokyo), the case numbers do not agree so the active case numbers are not consistent. 

Since you've reported it, I've been meaning to figure out how to fix this. I've put in a manually applied correct to the data so that the active cases numbers take into account this discrepency. 

I'll leave this issue open for the moment until we have a more permanent fix.
